src/messages/styles/popupMessageHub.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../styles/settings';
+
+.popup-message-container {
+  z-index: $z-index-high;
```

Comment:
Still in a issue here with the content boxes on the home page having a higher z-index

Reply:
Maybe I'm looking at the wrong thing, but it seems ok?

![image](https://user-images.githubusercontent.com/3329765/80082878-dc677d80-854c-11ea-8458-b4a1c356db4b.png)

Reply:
Somehow my setup doesn't work anymore so can't confirm right now. But yeah, the messages were going below those boxes. If what you see now is fine I guess I had old code and you can resolve that.